h/security/identity.py

```diff
@@ -1,11 +1,20 @@
 """Data classes used to represent authenticated users."""
 
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 from typing import List, Optional, Self
 
 from h.models import AuthClient, Group, User
 
 
+@dataclass
+class LongLivedMembership:
```

Comment:
I don't think I quite understand these, there's some mention about WS using detached objects at some point but I not sure I understand why. In any case this follows the existing pattern, no need to untangle that now.

Reply:
I think this is the origin of it, IIRC: https://hypothes-is.slack.com/archives/C01RAT0PE72/p1632746730000100